src/vue-components/modals/importModal.vue

```diff
@@ -152,10 +152,37 @@
                 if (this.options.resetBeforeImport) {
                     await dataService.markCurrentConfigAsBackedUp();
                 }
-                MainVue.showProgressBar(100);
-                if (this.reloadFn) {
-                    this.reloadFn();
-                }
+
+                // Wait for progress bar to close, then show success message
+                setTimeout(() => {
+                    let importedGridsCount = this.importData.grids ? this.importData.grids.length : 0;
+                    let importedDictsCount = this.options.importDictionaries && this.importData.dictionaries ? this.importData.dictionaries.length : 0;
+
+                    console.log('Showing success modal with counts:', importedGridsCount, importedDictsCount);
+
+                    let items = [];
+                    if (importedGridsCount > 0) {
+                        items.push(`${importedGridsCount} grid(s) imported`);
+                    }
+                    if (importedDictsCount > 0) {
+                        items.push(`${importedDictsCount} dictionaries imported`);
+                    }
+
+                    MainVue.showSuccessModal({
+                        header: i18nService.t('importSuccessful'),
+                        items: items,
+                        autoCloseDuration: 2000,
```

Comment:
I would'nt do a auto close duration here. I think if we decide on a blocking  full screen success message it should be closed manually by the user. For me it was irritating that there is a full screen modal which disappeared before I completely reckognized it's content. And we cannot know how long users will need to fully reckognize the content.